bl/validation/k8sValidator.go

```diff
@@ -182,7 +182,7 @@ func (val *K8sValidator) validateResource(filepath string) (bool, []error, *vali
 }
 
 func (val *K8sValidator) isNetworkError(errorString string) bool {
-	return strings.Contains(errorString, "no such host") || strings.Contains(errorString, "connection refused")
+	return strings.Contains(errorString, "no such host") || strings.Contains(errorString, "connection refused") || strings.Contains(errorString, "i/o timeout")
```

Comment:
Suggestion:
networkErrStr = ["no such host","connection refused","i/o timeout"] and iterate over it

Reply:
this will force me to add another function to iterate 
https://stackoverflow.com/questions/15323767/does-go-have-if-x-in-construct-similar-to-python
go doesn't have `arr.includes` logic :)

Reply:
I think that iterating twice is good here. if we iterate the same array.

Reply:
done :)